device-types/Cisco/Meraki-MR57.yaml

```diff
@@ -2,7 +2,6 @@
 manufacturer: Cisco
 model: MR57
 slug: MR57
-u_height: 0
```

Comment:
These are not rack mountable to my knowledge. We should leave `u_height: 0` defined